web/ui/mantine-ui/src/pages/query/ExplainViews/Aggregation.tsx

```diff
@@ -45,7 +45,21 @@ const describeAggregationType = (
         </>
       );
     case "bottomk":
-      return "returns the bottom K series by value";
+      if (param === null) {
+        throw new Error(
+          "encountered bottomk() node without K parameter"
+        );
+      }
+      if (param.type !== "numberLiteral") {
```

Comment:
This would break (exotic) usage cases that currently work where the parameter is not a literal, like https://demo-new.promlabs.com/query?g0.expr=topk%28scalar%28count%28demo_memory_usage_bytes%29+%2F+10%29%2C+demo_memory_usage_bytes%29&g0.show_tree=1&g0.tab=explain&g0.range_input=1h&g0.res_type=auto&g0.res_density=medium&g0.display_mode=lines&g0.show_exemplars=0

Maybe just fall back to the old explanation in this case?